src/rust/engine/process_execution/src/lib.rs

```diff
@@ -153,3 +158,39 @@ impl CommandRunner for BoundedCommandRunner {
     self.inner.1.with_acquired(move || inner.0.run(req))
   }
 }
+
+#[cfg(test)]
+mod tests {
+  use super::ExecuteProcessRequest;
+  use std::collections::hash_map::DefaultHasher;
+  use std::collections::{BTreeMap, BTreeSet};
+  use std::hash::Hash;
+
+  #[test]
+  fn execute_process_request_equality() {
```

Comment:
It's probably worth also adding a test which shows that if something other than `description` changes, the requests _are_ equal and do hash the same, as we're messing with that generality.